src/state/experiments/selectors.js

```diff
@@ -29,6 +33,11 @@ export const getFilteredExperimentsByDate = (state) => {
         return false;
       }
     }
+    if (status) {
+      if (mapStatusToInts[status.toLowerCase()] > mapStatusToInts[experimentStatus.toLowerCase()]) {
```

Comment:
You don't need to look this up everytime, it doesn't change on each iteration?

Reply:
Also good, mapping them onto integers and then comparing is exactly the right strategy 👍

Reply:
this nested if statement can probably be folded into the one above

Reply:
Ah yes good points, fixedddd 🍨